### ISSUE TITLE
Display part of the copied sequence in the notification toast

### ISSUE THREAD

**Issue by dgruano** [manulera/ShareYourCloning_frontend#217]
When a sequence is copied in the "Sequence" tab, a notification toast appears saying "Selection Copied".

It would be helpful for users to see the start/end/both of the sequence to make sure they copied the appropriate frame or strand.

**Comment by manulera** [comment]
Yes, I had a look at Benchling and it is quite nice that they display that. As I mentioned in #216, I think it would also be nice to include "Copied top strand sequence" or bottom, so that people are sure what was copied.

To do that with OVE, see [here](https://github.com/TeselaGen/tg-oss/tree/master/packages/ove#editorprops), the `onCopied` property of the editor could be overriden. In fact, we should probably overload those toasts with our own because they also do other things. For instance, I noticed that when copying, the y-scroll position snaps to top. I don't think this is something SYC is doing, but worth double-checking.
